src/ship_receiver_plugin.cpp

```diff
@@ -196,10 +195,15 @@ class ship_receiver_plugin_impl : std::enable_shared_from_this<ship_receiver_plu
          channels::native_trx native_trx = {trace.id, trace.cpu_usage_us, trace.elapsed};
          const auto& actions = trace.action_traces;
          //SILK_DEBUG << "Appending transaction ";
+         auto it = std::find_if(actions.begin(), actions.end(), [&](const auto& act) -> bool {
```

Comment:
Maybe later on we can find some optimization to quickly know that we are at an EOS EVM version greater than 0 and then simply avoid doing this additional linear scan through the actions of the transaction trace.